kubernetes_platform/python/kfp/kubernetes/config_map.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2023 The Kubeflow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict
+
+from google.protobuf import json_format
+from kfp.dsl import PipelineTask
+from kfp.kubernetes import common
+from kfp.kubernetes import kubernetes_executor_config_pb2 as pb
+
+
+def use_config_map_as_env(
+    task: PipelineTask,
+    config_map_name: str,
+    config_map_key_to_env: Dict[str, str],
+) -> PipelineTask:
+    """Use a Kubernetes ConfigMap as an environment variable as described in
+    https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#define-container-environment-variables-using-configmap-data.
```

Comment:
Can we style the link using RST for clean rendering in the reference docs?

Reply:
done, thanks